pkgs/tools/system/logrotate/default.nix

```diff
@@ -14,27 +14,21 @@ stdenv.mkDerivation rec {
   };
 
   # Logrotate wants to access the 'mail' program; to be done.
-  patchPhase = ''
-    sed -i -e 's,[a-z/]\+gzip,${gzip}/bin/gzip,' \
-           -e 's,[a-z/]\+gunzip,${gzip}/bin/gunzip,' configure.ac
-
-    ${lib.optionalString (mailutils != null) ''
-    sed -i -e 's,[a-z/]\+mail,${mailutils}/bin/mail,' configure.ac
-    ''}
-  '';
-
-  autoreconfPhase = ''
-    ./autogen.sh
-  '';
+  configureFlags = [
```

Comment:
Maybe we should add strictDeps?

Reply:
@SuperSandro2000 sorry, busy day... I'll have to look into `strictDeps` as I haven't worked with them yet. Thanks for the suggestion and merge!